Aquarius/Src/Aquarius/Events/MouseEvent.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include "Event.h"
+
+#include <GLFW/glfw3.h>
+
+
+namespace Aquarius {
+
+    class MouseMovedEvent : public Event
+    {
+    public:
+        MouseMovedEvent(float xCoordinate, float yCoordinate)
+                : m_XCoordinate(xCoordinate), m_YCoordinate(yCoordinate) {}
```

Comment:
I think just m_x and m_y would suffice? Or x_pos? x_coord? xCoordinate seems a little bit verbose.

For me to get the x/y of a mouse, I would need to call:
```
mouseMovedEvent.getXCoordinate()
mouseMovedEvent.getYCoordinate()
```

Reply:
I would also add a getter that returns a glm::vec2 or std::pair of the (x,y) coordinates